snomed/com.b2international.snowowl.snomed.datastore.tests/src/com/b2international/snowowl/snomed/core/ecl/SnomedEclEvaluationRequestTest.java

```diff
@@ -725,6 +724,25 @@ public void refinementWithGroupConjunction() throws Exception {
 		assertEquals(expected, actual);
 	}
 	
+	@Test
+	public void refinementGroupCardinality() throws Exception {
```

Comment:
Please choose a better method name, there is no group cardinality in this test case!